GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -244,9 +266,107 @@ gdjs.LayerPixiRenderer.prototype.enableEffect = function(name, value) {
  * @param {string} name The effect name
  * @return {boolean} true if the filter is enabled
  */
-gdjs.LayerPixiRenderer.prototype.isEffectEnabled = function(name) {
+gdjs.LayerPixiRenderer.prototype.isEffectEnabled = function (name) {
   var filter = this._filters[name];
   if (!filter) return false;
 
   return gdjs.PixiFiltersTools.isEffectEnabled(filter);
 };
+
+/**
+ * Updates the render texture, if it exists. 
+ * Also, render texture is cleared with a specified clear color.  
+ * Not to be called outside of updateTime method in most cases.
+ */
+gdjs.LayerPixiRenderer.prototype.updateRenderTexture = function () {
```

Comment:
Make this private with an _

Reply:
Should I annotate it with a `@private` as well?

Reply:
Yes you can!